README.md

```diff
@@ -8,6 +8,30 @@ Container Platform.
 The suite is provided here in part so that CNF Developers can use the suite to test their CNFs readiness for
 certification.  Please see "CNF Developers" below for more information.
 
+## Container
+
+### Running
+
+To run the tests in the container, use the following command. There are several required arguments:
+
+* `-k` gives a path to the kube config file to be used by the container to authenticate with the cluster.
+* `-t` gives the local directory that contains tnf config files set up for the test.
+* `-o` gives the local directory that the test results will be available in once the container exits.
+* Finally, the specs to be run must be specified.
+
+```shell-script
+./run-container.sh -k ~/.kube/config -t ~/tnf/config -o ~/tnf/output diagnostic generic
```

Comment:
We may want to consider making diagnostic enabled by default, and adding a flag to disable.  That said, that is work for a future version.